### PR TITLE
feat(devCDN): fasitfy spike

### PR DESCRIPTION
**Pull request by JAdshead**
## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
     "yargs": "^15.3.1"
   },
   "devDependencies": {
-    "@americanexpress/one-app-dev-cdn": "^3.3.1",
+    "@americanexpress/one-app-dev-cdn": "file:../one-app-dev-cdn/americanexpress-one-app-dev-cdn-3.3.2.tgz",
```
Comment (Matthew-Mallimo) [inline package.json:130]: Just pointing this out for the future so it is not forgotten

**Review by Matthew-Mallimo — COMMENTED** [review]
Any noticeable improvements with the switch ?

**Comment by JAdshead** [comment]
> Any noticeable improvements with the switch ?

it feels like the app starts faster but would need to measure to be sure

**Comment by 10xLaCroixDrinker** [comment]
Superceded by #861

**Pull request closed without merging**